insights/configtree/dictlike.py

```diff
@@ -0,0 +1,114 @@
+from insights.configtree import PushBack, parse_string, typed
```

Comment:
Please add this module to `docs/api_index.rst` and add docstrings to include it in the documentation.